### PR TITLE
added text editor 

### PR DESCRIPTION
**Pull request by narharikale**
- added text editor

### CODE REVIEW

File: src/frontend/components/sidebar/sidebar.css [hunk 1]
```diff
@@ -36,6 +36,7 @@
         background-color: var(--color-background-accent);
         height:87%;
         box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
```
Comment (ayush65) [inline src/frontend/components/sidebar/sidebar.css:38]: you can use variables for the rgba color

File: src/frontend/components/textEditor/TextEditor.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                  }  
+              </div>
+              <div>
+                <button className='notesy-btn notesy-primary-btn'>Add</button>
+              </div>
+            </div>
+            
+        </div>  
+       
+
```
Comment (ayush65) [inline src/frontend/components/textEditor/TextEditor.jsx:64]: remove extra spaces

**Review by ayush65 — APPROVED** [review]
Code Looks clean 
class Names are self Explanatory

File: src/frontend/components/textEditor/TextEditor.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  const [value, setValue] =  useState("");
+        console.log(value);
+
+  return (
+    <div className="main-textEditor">
+        <div className='text-editor-container d-flex gap-1 flex-column'>
+            <div>
+                <input type="text" placeholder='Title' className='text-editor-title font-size-md' />
+            </div>
+            <ReactQuill modules={modules} onChange={ setValue }  theme="snow" placeholder='Take a note...' />
```
Comment (vijaytembugade) [inline src/frontend/components/textEditor/TextEditor.jsx]: Set value and defaultValue

File: src/frontend/components/textEditor/TextEditor.jsx (hunk 2 above)
Comment (narharikale) [reply src/frontend/components/textEditor/TextEditor.jsx:64]: done

**Review by vijaytembugade — APPROVED** [review]

File: src/frontend/components/sidebar/sidebar.css (hunk 1 above)
Comment (narharikale) [reply src/frontend/components/sidebar/sidebar.css:38]: i will try

File: src/frontend/components/textEditor/TextEditor.jsx (hunk 3 above)
Comment (narharikale) [reply src/frontend/components/textEditor/TextEditor.jsx]: fixed

**Pull request merged**